### PR TITLE
add `CHECK_NAME_DUPLICATES` feature

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Adds a new feature `CHECK_NAME_DUPLICATES` in the config which disables name-based duplicate checking. This is helpful when your GTFS has common names in it.

### CODE REVIEW

File: handler.ts [hunk 1]
```diff
@@ -128,7 +133,12 @@ export const makeGeocoderRequests = async (
   >(
     (prev, cur, idx) => {
       if (idx === 0) return cur
-      return mergeResponses({ customResponse: cur, primaryResponse: prev })
+      return mergeResponses(
+        { customResponse: cur, primaryResponse: prev },
+        // Default to false
```
Comment (binh-dam-ibigroup) [inline handler.ts:138]: Did you mean to say: "Default to true"?

File: handler.ts [hunk 2]
```diff
@@ -128,7 +133,12 @@ export const makeGeocoderRequests = async (
...
+        CHECK_NAME_DUPLICATES !== 'false'
+        // convertQSPToGeocoderArgs(event.queryStringParameters)?.focusPoint
```
Comment (binh-dam-ibigroup) [inline handler.ts:140]: Remove commented code.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM, please clean up the comments.

File: handler.ts (hunk 2 above)
Comment (miles-grant-ibigroup) [reply handler.ts]: Good catch thanks

File: __tests__/util.test.ts [hunk 3]
```diff
@@ -112,13 +112,15 @@ describe('response merging', () => {
         customResponse: CUSTOM_RESPONSE,
         primaryResponse: GEOCODE_EARTH_RESPONSE
       },
+      true,
```
Comment (daniel-heppner-ibigroup) [inline __tests__/util.test.ts:115]: can we get a test that tests this with `false`?

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]
How hard would it be to add a test for this functionality?

File: __tests__/util.test.ts (hunk 3 above)
Comment (miles-grant-ibigroup) [reply __tests__/util.test.ts:115]: I tried but couldn't do it in any way that made sense. This test tests for something completely different. We could add an entire new test case but this is a really weird edge case so I'm tempted to add a TODO here instead?

Comment (daniel-heppner-ibigroup) [reply __tests__/util.test.ts:115]: I'd be fine with a TODO.

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM! You can add the todo if you want but I think it probably doesn't matter. We'll probably never get to it anyway.
